File: app/src/main/java/com/starostinvlad/fan/SearchScreen/SearchFragmentContract.java
package com.starostinvlad.fan.SearchScreen;

import com.starostinvlad.fan.BaseMVP.MvpView;
import com.starostinvlad.fan.GsonModels.News;

import java.util.List;

public interface SearchFragmentContract extends MvpView {
    void showLoading(boolean show);

    void fillList(List<News> arr);

    void showMessage(String message);
}

File: app/src/main/java/com/starostinvlad/fan/SearchScreen/SearchFragmentContract.kt
package com.starostinvlad.fan.SearchScreen

import com.starostinvlad.fan.BaseMVP.MvpView
import com.starostinvlad.fan.GsonModels.News

interface SearchFragmentContract : MvpView {
    fun showLoading(show: Boolean)
    fun fillList(arr: MutableList<News>)
    fun showMessage(message: String?)
}